goto/the_real_goto.py

```diff
@@ -9,20 +9,7 @@
 from .gotomagic import text
 from .gotomagic.magic import GotoMagic
 
-from .commands import\
-    usage,\
-    add,\
-    update,\
-    rm,\
-    show,\
-    copy,\
-    list,\
-    subl,\
-    vscode,\
-    intellij,\
-    open,\
-    cd,\
-    default
+from . import commands
```

Comment:
Hmmm... I don't like removing the explicit imports 👎 but it is not a dealbreaker.